test/go.mod

```diff
@@ -1,27 +1,27 @@
 module github.com/metal3-io/cluster-api-provider-metal3/test
 
-go 1.22
+go 1.22.3
 
 require (
-	github.com/docker/docker v26.1.4+incompatible
+	github.com/docker/docker v27.0.1-rc.1+incompatible
 	github.com/jinzhu/copier v0.4.0
 	github.com/metal3-io/baremetal-operator/apis v0.6.1
-	github.com/metal3-io/cluster-api-provider-metal3/api v0.0.0
+	github.com/metal3-io/cluster-api-provider-metal3/api v0.0.0-00010101000000-000000000000
```

Comment:
Where did this come from?

Reply:
Strange! didn't notice it, I wonder what caused it